src/components/ButtonGroup/PButtonGroup.vue

```diff
@@ -3,9 +3,9 @@
     <template v-for="button in options" :key="button.label">
       <PButton
         :disabled="button.disabled"
-        :size="size"
-        :inset="button.value !== modelValue"
+        :small="small"
         class="p-button-group__button"
+        :class="{ 'p-button-group__button--selected': button.value === modelValue }"
```

Comment:
We should avoid computed objects in the template cause it's tough to extend/maintain. I'd put this into a computed variable in the script instead